### DISCUSSION TITLE
Change of project domain

### DISCUSSION THREAD

**Discussion by aazbeltran** [Laravel-Backpack/community-forum#652] [Q&A (Help)] [answered]
Hi, I already have a license and I plan to start a new project but I still don't have a name for this project at this stage, because for the purchase of the PRO license, I need a domain name, I would like to know if it is possible to use a temporary domain and then request a change or how can I proceed in this situation?

**Comment by karandatwani92** [comment, marked as answer]
Hey @aazbeltran
Yes, feel free to use a project name for now. Later, simply ask to change the domain. hello@backpackforlaravel.com

@tabacitu will help you out.✌️
